src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+    class="content-page">
+      <h1>CALENDAR</h1>
+      <div class="calendar calendar-page--tue calendar-page--month-28">
```

Comment:
when modifier is set to `calendar-page--tue`, first day of the calendar is supposed to be tuesday, it doesn't look like that now. Also when I change this modifier to `calendar-page--wed`, first day of the month is supposed to be wednesday. please fix it

Reply:
done